perception/multi_object_tracker/src/tracker/model/tracker_base.cpp

```diff
@@ -34,23 +35,74 @@ Tracker::Tracker(
   std::mt19937 gen(std::random_device{}());
   std::independent_bits_engine<std::mt19937, 8, uint8_t> bit_eng(gen);
   std::generate(uuid_.uuid.begin(), uuid_.uuid.end(), bit_eng);
+
+  // Initialize existence probabilities
+  existence_probabilities_.resize(channel_size, 0.0);
+}
+
+void Tracker::initializeExistenceProbabilities(
+  const uint & channel_index, const float & existence_probability)
+{
+  existence_probabilities_[channel_index] = 0.8 + 0.2 * existence_probability;
+  total_existence_probability_ = existence_probability;
 }
 
 bool Tracker::updateWithMeasurement(
   const autoware_auto_perception_msgs::msg::DetectedObject & object,
-  const rclcpp::Time & measurement_time, const geometry_msgs::msg::Transform & self_transform)
+  const rclcpp::Time & measurement_time, const geometry_msgs::msg::Transform & self_transform,
+  const uint & channel_index)
 {
-  no_measurement_count_ = 0;
-  ++total_measurement_count_;
+  // Update existence probability
+  {
+    float existence_probability_from_object = object.existence_probability;
+    no_measurement_count_ = 0;
+    ++total_measurement_count_;
+
+    // existence probability on each channel
+    const double delta_time = (measurement_time - last_update_with_measurement_time_).seconds();
+    const double decay_rate = 5.0 / 10.0;
+
+    const float gain = 0.4;
+    const float probability_detected = 0.99;
+    // existence_probabilities_[channel_index] = existence_probability_from_object;
+    existence_probabilities_[channel_index] =
+      gain * probability_detected + (1 - gain) * existence_probabilities_[channel_index];
+
+    for (size_t i = 0; i < existence_probabilities_.size(); ++i) {
+      if (i == channel_index) {
+        continue;
+      }
+      existence_probabilities_[i] *= std::exp(-decay_rate * delta_time);
+    }
+
+    // total existence probability - object is detected
+    total_existence_probability_ +=
```

Comment:
[imho]
Hmm, with this update, existence_probability just increases.
It may not bad, but how about implementing bayes rule?


```math
p' = \frac{p'*p}{p'*p + (1-p')*(1-p)} 
```
while p' is trackers probability and p is input.

```cpp
double updateProbability(double prior, double likelihood) {
    return (prior * likelihood) / (prior * likelihood + (1 - prior) * (1 - likelihood));
}
```


Here is sample update results:

| Prior Probability p' | Observation Probability p | Initial Formula Result | Bayesian Update Result |
|----------------------|---------------------------|------------------------|------------------------|
| 0.5                  | 0.8                       | 0.9                    | 0.8                    |
| 0.8                  | 0.5                       | 0.9                    | 0.6667                 |

Reply:
Thank you for your advise.
Method of the  probability update is fixed. 

Object confidentiality will be revised later and the probability tracking result will be used.
Detailed numbers will be adjusted. 

https://github.com/autowarefoundation/autoware.universe/pull/6820/commits/f218f84a943278b28f2ac55c0cf8f142b108e376